### PR TITLE
add EXPIRE and EXPIREAT passive strategy implementation and tests

### PR DESCRIPTION
**Pull request by axblueblader**
- command will store timestamp of expiration in database using key + special postfix
- actual delete happens in separate thread and only triggered when the key is queried after expiration time (hence passive)

### CODE REVIEW

File: server/src/main/java/dev/keva/server/command/impl/key/manager/ExpirationManager.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import dev.keva.ioc.annotation.Autowired;
+import dev.keva.ioc.annotation.Component;
+import dev.keva.store.KevaDatabase;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Component
+public class ExpirationManager {
+    private static final byte[] EXP_POSTFIX = new byte[]{(byte) 0x7f, (byte) 0x2f, (byte) 0x61, (byte) 0x74};
```
Comment (tuhuynh27) [inline server/src/main/java/dev/keva/server/command/impl/key/manager/ExpirationManager.java:14]: @axblueblader Better should have a comment here describe about the bytes value

**Review by tuhuynh27 — APPROVED** [review]
LGTM

**Comment by tuhuynh27** [comment]
Also @axblueblader, when you add a new command, you can [update docs](https://github.com/keva-dev/keva/blob/master/docs/src/guide/overview/commands.md) too

File: server/src/main/java/dev/keva/server/command/impl/string/Get.java [hunk 2]
```diff
@@ -15,15 +16,23 @@
...
+    public Get(KevaDatabase database, ExpirationManager expirationManager) {
         this.database = database;
+        this.expirationManager = expirationManager;
     }
 
     @Execute
     public Reply<?> execute(byte[] key) {
-        val got = database.get(key);
-        return got == null ? BulkReply.NIL_REPLY : new BulkReply(got);
+        boolean expirable = expirationManager.isExpirable(key);
```
Comment (duongcongtoai) [inline server/src/main/java/dev/keva/server/command/impl/string/Get.java:29]: should we validate for set as well? like key is not allowed to have internal postfix.

Comment (axblueblader) [reply server/src/main/java/dev/keva/server/command/impl/string/Get.java:29]: postfix has a special character, so it's unlikely to be accidentally set

**Comment by tuhuynh27** [comment]
Latest changes LGTM, good work @axblueblader

**Pull request merged**